doc: improve instructions for building on cmd line (#5276)
nrfconnect/sdk-nrf: doc/nrf/gs_installing.rst

      This file is loaded automatically when you run the above command.
      See :ref:`zephyr:env_vars_zephyrrc` for more information.

Install nrfjprog
================

You must also make sure that nrfjprog (part of the `nRF Command Line Tools`_) is installed and its path is added to the environment variables.

> The west command programs the development kit by using nrfjprog by default.

Reviewer: ```suggestion
The west command uses nrfjprog to program the development kit by default.
```
I think that would be a more readable sentence.

Author: Thanks! I added this change.

For more information on nrfjprog, see `Programming SoCs with nrfjprog`_.

.. note::

>    If you installed |NCS| through the Toolchain Manager, you must install nrfjprog separately using pip.

Reviewer 2: The _nRF Command Line Tools_ is all that is needed. You don't need to install this in addition. I tried to install `pynrfjprog` with pip, but would still get an error that `nrfjprog` is not installed. So I think it's safe to remove this note.

   .. parsed-literal::
      :class: highlight

      pip install pynrfjprog

>    After installing nrfjprog, continue to the steps listed in :ref:`gs_programming_cmd`.

Reviewer 3: ```suggestion
   After installing nrfjprog, you can start :ref:`gs_programming_cmd`.
```

Author: Thanks! I added this change.
